### PR TITLE
Kubernetes integration 

### PR DESCRIPTION
**Pull request by gkeesh7**
Adding Deployment.yaml and Service.yaml file to make the Service run on kubernetes cluster.

### CODE REVIEW

File: logic/expiry_logic.go [hunk 1]
```diff
@@ -6,8 +6,10 @@ import (
 	"url-shortner/storage/dao"
 )
 
+const _deleteQuery = "DELETE from redirection WHERE expiry < NOW()"
```
Comment (sameer-bishnoi) [inline logic/expiry_logic.go:9]: Why it's a constant, not a variable?

**Review by sameer-bishnoi — APPROVED** [review]
Always leave 1 blank line at the end of `yaml` file. Just 1 minor comment, rest LGTM 👍

File: logic/expiry_logic.go (hunk 1 above)
Comment (gkeesh7) [reply logic/expiry_logic.go:9]: This delete query should not change over the course of the running App hence a constant

Comment (sameer-bishnoi) [reply logic/expiry_logic.go:9]: Okay, bro. 
JFYI: Constant are always written in Upper case 🙂

Comment (gkeesh7) [reply logic/expiry_logic.go:9]: I use this styleguide for reference [Style Guide](https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_)

**Pull request merged**